### PR TITLE
Adding the method `Flores::PKI.generate` to return a valid Certificate and a key

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: lib/flores/pki.rb [hunk 1]
```diff
@@ -21,12 +21,38 @@
...
+    # Generate a random serial number for a certificate.
+    def random_serial
+      # RFC5280 (X509) says:
+      # > 4.1.2.2.  Serial Number 
+      # > Certificate users MUST be able to handle serialNumber values up to 20 octets
+      Flores::Random.integer(1..9).to_s + Flores::Random.iterations(0..19).collect { Flores::Random.integer(0..9) }.join
+    end
+
+    # Generate a valid certifcate with sane random values
+    # return [Certicate, OpenSSL::Pkey::RSS]
```
Comment (jordansissel) [inline lib/flores/pki.rb]: Can we document the opts?

Comment (jordansissel) [inline lib/flores/pki.rb]: Pkey::RSA, right?

**Comment by jordansissel** [comment]
Added some inline comments. Otherwise mostly LGTM. Haven't tested yet.

File: lib/flores/pki.rb (hunk 1 above)
Comment (ph) [inline lib/flores/pki.rb]: but RSS is so 2004

**Comment by ph** [comment]
Updated with the changes from the code review.
Nice documentation, you are planning to use yard?

**Comment by jordansissel** [comment]
YARD probably; I mostly wish for making rubydoc.info show the docs well.

**Comment by jordansissel** [comment]
New flores 0.0.6 published

**Pull request merged**